cmd/ko/commands.go

```diff
@@ -70,7 +71,7 @@ func addKubeCommands(topLevel *cobra.Command) {
 		Run: func(cmd *cobra.Command, args []string) {
 			// TODO(mattmoor): Use io.Pipe to avoid buffering the whole thing.
```

Comment:
You don't even need a pipe. Set up the exec.Command before here and pass `kubectlCmd.Stdin` to `resolveFilesToWriter`